costs/diff/diff_route.go

```diff
@@ -107,13 +107,13 @@ func makeElasticSearchRequest(ctx context.Context, parsedParams esQueryParams, u
 func getDiffData(request *http.Request, a routes.Arguments) (int, interface{}) {
 	user := a[users.AuthenticatedUser].(users.User)
 	parsedParams := esQueryParams{
-		accountList:       []int{},
+		accountList:       []string{},
 		dateBegin:         a[diffQueryArgs[1]].(time.Time),
 		dateEnd:           a[diffQueryArgs[2]].(time.Time),
 		aggregationPeriod: a[diffQueryArgs[3]].(string),
 	}
 	if a[diffQueryArgs[0]] != nil {
-		parsedParams.accountList = a[diffQueryArgs[0]].([]int)
+		parsedParams.accountList = a[diffQueryArgs[0]].([]string)
```

Comment:
This creates a panic: diffQueryArgs[0] uses routes.AwsAccountsOptionalQueryArg which is a []int, you cannot cast it to []string